src/devices/src/virtio/block/device.rs

```diff
@@ -413,13 +396,54 @@ mod tests {
     #[test]
     #[allow(clippy::cognitive_complexity)]
     fn test_virtio_device() {
-        let mut b = default_block(true);
+        let mut b = default_block();
 
         // Test `device_type()`.
         {
             assert_eq!(b.device_type(), TYPE_BLOCK);
         }
 
+        // Test `get_queues()`.
+        {
+            // After initialization, the `queues` field should contain
+            // only one empty queue.
+            let queues = b.get_queues();
+            assert_eq!(queues.len(), 1);
+            assert_eq!(queues[0].actual_size(), 0);
+        }
+
+        // Test `get_queue_events()`.
```

Comment:
What are you trying to test here? This test doesn't seem to test the functionality of virtio block.

Reply:
I removed these tests as they don't seem useful in checking the functionality of virtio.